pkg/controller/managedresources/merger.go

```diff
@@ -25,6 +25,12 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
+const (
+	descriptionAnnotation     = "resource-manager.gardener.cloud/description"
```

Comment:
WDYT about changing the annotation key to `resources.gardener.cloud/description` to match the CRD's API group?
```suggestion
	descriptionAnnotation     = "resources.gardener.cloud/description"
```